contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+
+
+Install
+-------
+
+The easiest method for installing Py-ART is to use the conda packages from
+the latest release.  To do this you must download and install 
+`Anaconda <http://continuum.io/downloads>`_ or 
+`Miniconda <http://continuum.io/downloads>`_.  
+Then use the following command in a terminal or command prompt to install
+the latest version of Py-ART::
+
+    conda install -c conda-forge arm_pyart
+
+To update an older version of Py-ART to the latest release use::
+
+    conda update -c conda-forge arm_pyart
+
+Resources
+=========
+
+Pyart:
+
+- https://github.com/openradar/AMS-Short-Course-on-Open-Source-Radar-Software
+- https://github.com/EVS-ATMOS/pyart_short_course
+- https://www.youtube.com/watch?v=diiP-Q3bKZw
+- http://arm-doe.github.io/pyart/dev/auto_examples/index.html
+
+Git:
+
+- https://git-scm.com/book/en/v2
+- https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html
+
+
+Proper Code Style
+=================
+
+Py-ART follows pep8 coding standards. To make sure your code follows the pep8
+style, you can use a variety of modules that can check for you. Two popular
+pep8 check modules are pycodestyle and pylint.
+
+For more on pep8 style:
+
+- https://www.python.org/dev/peps/pep-0008/
+
+To install pycodestyle::
+
+        pip install pycodestyle
+
+To use pycodestyle::
+
+        pycodestyle filename
+
+pylint is also useful to check for pep8 compliance and should already be
+installed with python 3.5.
+
+To get a detailed pylint report::
+
+        pylint filename
+
+If you want to just see what line number and the issue, just use::
+
+        pylint -r n filename
+
+
+Python File Setup
+=================
+
+In a new .py file, the top of the code should have the function's location,
+sphinx comments for template configuration, and public and private functions
+within the .py file. Public functions are listed first and then private
+functions with and underscore in front of the function. A space is needed
+between the last function and the closing docstring quotation.
+
+For example:
+
+.. code-block:: python
+
+        """
+	pyart.retrieve.velocity_azimuth_display
+	=======================================
+	
+	Retrieval of VADs from a radar object.
+
+	.. autosummary::
+    	    :toctreeL generated/
+    	    :template: dev_template.rst
+
+	    velocity_azimuth_display
+            _inverse_dist_squared
+            _Average1D
+
+        """
+
+Following the introduction code, modules are then added. Main imports come
+first, followed by 'from imports' and user's own modules.
+
+For example:
+
+.. code-block:: python
+
+        import numpy as np
+        import numpy.ma as ma
+        from scipy.interpolate import interp1d
+
+        from ..core import HorizontalWindProfile
+
+Following the main function def line, but before the code within it, a doc
+string is needed to explain arguments, returns, references if needed, and
+other helpful information.
```

Comment:
These documentation standard are follow the NumPy documentation style, https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

Would be good to include this note and link here.